x-pack/plugins/uptime/server/lib/saved_objects/synthetics_monitor.ts

```diff
@@ -32,6 +32,9 @@ export const syntheticsMonitor: SavedObjectsType = {
       tags: {
         type: 'keyword',
       },
+      secrets: {
```

Comment:
It's not necessary to add this here if we aren't querying by secrets. Will remove.